src/test/java/se/lth/base/server/rest/UserResourceTest.java

```diff
@@ -128,45 +132,15 @@ public void createUserAsUser() {
                 .post(Entity.json(""), Void.class); // Include response type to trigger exception
     }
 
-    @Test(expected = ForbiddenException.class)
```

Comment:
Why is this deleted?

Reply:
The only thing i remember changing in the test files was that i added my test and removed the test that checked so that a user couldnt remove themselves. Because i know this test was removed in another issue but for some reason it was back and failing since it was now possible to delete yourself. I didn't delete this test or change any others, at least not intentionally.